manifests/init.pp

```diff
@@ -130,12 +139,14 @@
       $svc_ensure  = $service_ensure
       $svc_enable  = $service_ensure ? { 'running' => true, 'stopped' => false }
       $file_ensure = 'present'
+      $srv_manage  = $manage_service
     }
     'absent', 'purged': {
       $pkg_ensure  = $ensure
       $svc_ensure  = 'stopped'
       $svc_enable  = false
       $file_ensure = 'absent'
+      $srv_manage  = false
```

Comment:
Does this imply that the service will never be stopped unless the packaging has an uninstall script that handles it?